server/server.js

```diff
@@ -1,16 +1,43 @@
 const express = require('express');
+const querystring = require('querystring');
+const bodyParser = require('body-parser');
+const dbConnection = require('../database/db_connection');
```

Comment:
I don't think this should be here either. All database stuff should be handled outside of your handling/routing files.